src/parser/permit-generation-module.ts

```diff
@@ -149,34 +153,35 @@ export class PermitGenerationModule implements Module {
    * This method is meant to be called before the final permit generation.
    * @param result Result object
    * @param erc20RewardToken ERC20 address of the reward token
+   * @param env The program environment
    * @returns Result object
    */
-  async _applyFees(result: Result, erc20RewardToken: string): Promise<Result> {
+  async _applyFees(result: Result, erc20RewardToken: string, env: EnvConfigType): Promise<Result> {
     // validate fee related env variables
-    if (!process.env.PERMIT_FEE_RATE || +process.env.PERMIT_FEE_RATE === 0) {
+    if (!env.PERMIT_FEE_RATE || +env.PERMIT_FEE_RATE === 0) {
```

Comment:
```suggestion
    if (!env.PERMIT_FEE_RATE || Number(env.PERMIT_FEE_RATE) === 0) {
```

This `+` syntax is janky and error-prone.